scrapy/pipelines/files.py

```diff
@@ -214,23 +214,33 @@ class FilesPipeline(MediaPipeline):
     """
 
     MEDIA_NAME = "file"
+    EXPIRES = 90
     STORE_SCHEMES = {
         '': FSFilesStore,
         'file': FSFilesStore,
         's3': S3FilesStore,
     }
+    DEFAULT_FILES_URLS_FIELD = 'file_urls'
+    DEFAULT_FILES_RESULT_FIELD = 'files'
 
```

Comment:
It does not fix the backward incompatibility (#1985). It must be `FILES_URLS_FIELD` and `FILES_RESULT_FIELD`, as it was before #1891. Look at this [line](https://github.com/scrapy/scrapy/blob/1.0/scrapy/pipelines/files.py#L169): `DEFAULT_*` is used just to populate the other one.

Here arises one question: should both (e.g. `DEFAULT_FILES_URLS_FIELD` and `FILES_URLS_FIELD`) be present now? Because one could have a custom Files/ImagesPipeline using one or another.

I think both should be present in order to totally fix the backward incompatibility:
```python
DEFAULT_FILES_URLS_FIELD = 'file_urls'
FILES_URLS_FIELD = DEFAULT_FILES_URLS_FIELD

DEFAULT_FILES_RESULT_FIELD = 'files'
FILES_RESULT_FIELD = DEFAULT_FILES_RESULT_FIELD = 'files'
```